src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -892,17 +893,16 @@
     </BoundTreeDefinition>
     <Compile Include="$(IntermediateOutputPath)BoundNodes.xml.Generated.cs" />
     <Compile Include="$(IntermediateOutputPath)ErrorFacts.Generated.cs" />
-    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Generated.cs" />
+    <Compile Include="$(IntermediateOutputPath)Syntax.xml.Syntax.Generated.cs" />
```

Comment:
Why not use the variables defined in GenerateCompilerInternals.targets? Otherwise there's some repeating of policy here.

(Also not sure why these aren't just in GenerateCompilerInternals.targets itself.)